aws-codeguruprofiler-profilinggroup/aws-codeguruprofiler-profilinggroup.json

```diff
@@ -26,7 +57,8 @@
   },
   "additionalProperties": false,
   "required": [
-    "ProfilingGroupName"
+    "ProfilingGroupName",
+    "Permissions"
```

Comment:
This seems potentially problematic -- what happens to existing customers that have cloudformation written without specifying permissions?

Reply:
It makes sense, I will update then the code for ```Permissions``` to not be required, thanks.